src/cascadia/TerminalApp/Profile.h

```diff
@@ -114,6 +114,9 @@ class TerminalApp::Profile final
     static std::tuple<winrt::Windows::UI::Xaml::HorizontalAlignment, winrt::Windows::UI::Xaml::VerticalAlignment> ParseImageAlignment(const std::string_view imageAlignment);
     static std::tuple<winrt::Windows::UI::Xaml::HorizontalAlignment, winrt::Windows::UI::Xaml::VerticalAlignment> _ConvertJsonToAlignment(const Json::Value& json);
 
+    static uint16_t _ParseFontWeight(const Json::Value& json);

```

Comment:
is there a better type we can use here? It will make our lives easier when we come through and use template-type-deducing json deserializers. Maybe a `FontWeight`?

Reply:
hmmmm. `Windows::UI::Font::FontWeights` might work. Let me try.